health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -0,0 +1,677 @@
+                setSelectedLevel(null);
+                setSelectedBoundary(null);
+              }}
+            />,
+          ]}
+          sortFooterChildren={true}
+          onClose={() => {
+            setShowPopUp(false);
+            setSelectedLevel(null);
+            setSelectedBoundary(null);
+          }}
+        ></PopUp>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)